src/rules/color-no-hex.ts

```diff
@@ -0,0 +1,50 @@
+import type { Rule } from 'eslint';
+/**
+ * Regular expression to match hex color values.
+ * Added capturing group to extract the hex value.
+ * Using 'u' flag for Unicode support.
+ */
+const hexColorRegex = /(#\p{Hex_Digit}{3,6})\b/u;
+
+/**
+ * Rule to prevent the use of hex color values.
+ */
+export const colorNoHex: Rule.RuleModule = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Prevent the use of hex color values.',
+      recommended: true,
+      url: 'https://github.com/your-plugin-url-here#color-no-hex',
```

Comment:
Need to update docs link

Reply:
Updated here https://github.com/MetaMask/eslint-plugin-design-tokens/pull/11/commits/00c5254d579ffb4ca299277c8d8bd9b5c3b2f0e0